### PR TITLE
Helm chart rename - Step #3

### PR DESCRIPTION
**Pull request by stevehipwell**
IMPORTANT - This needs to be merged after PR #457 and PR #459.

This PR will release the v0.19 chart with the new _descheduler_ name.

### CODE REVIEW

**Comment by damemi** [comment]
/hold
for steps 1 & 2

File: README.md
Comment (damemi) [inline README.md:14]: There's a lot of odd updates to the README.md in this PR, are these intentional or maybe an editor auto-formatting? Would like to undo these if possible

**Review by damemi — CHANGES_REQUESTED** [review]
looks like steps 1 & 2 worked out, the index.yaml is getting updated and the 1.18 release still the right naming scheme, so I think we can keep going with this PR

@stevehipwell thanks again for your help so far, could you just revert the changes in this PR to README.md? It seems like they were unintentional

**Comment by stevehipwell** [comment]
@damemi I made a change in the README but it looks like my formatter tidied it up too. I'm not by my machine at the moment so is it possible to take the change or do I need to update the PR? This is time sensitive as v0.19 will be missing from the hub so if I need to I can get back to my machine.

**Comment by damemi** [comment]
@stevehipwell I suppose those changes aren't a big problem, and it does correct some formatting that was messed up (like extra backtics in code blocks)

Since this is time sensitive, I'm fine merging the changes in formatting now too.
/hold cancel

Once this merges, please check that everything looks good on your end too and let us know. Thank you!

**Comment by stevehipwell** [comment]
@damemi this all looks good to me.

**Comment by stevehipwell** [comment]
@damemi it looks like the GitHub release action has failed for the v0.19 release.

**Comment by damemi** [comment]
@stevehipwell thanks, looks like a CI check was blocking it. Re-ran and it looks like it completed

**Comment by stevehipwell** [comment]
@damemi I think you either need to delete the `descheduler-helm-chart-0.19.2` tag and then re run the action or manually modify the _index.yaml_ file on the `gh-pages` branch; as it looks like the failure wasn't cleaned up correctly so resulted in a NOP.

**Comment by damemi** [comment]
@stevehipwell oh, that makes sense. I was able to delete the branch/release and re-run the job. The right version appears to be in index.yaml now: https://github.com/kubernetes-sigs/descheduler/blob/97a0ab9/index.yaml

and the release is back https://github.com/kubernetes-sigs/descheduler/releases/tag/descheduler-helm-chart-0.19.2

If you can double check that I got this right and I'm not missing anything else that would be great!

**Comment by stevehipwell** [comment]
@damemi I can now see the v0.19 chart on artifacthub.io so it looks good to me.

**Pull request merged**
